packages/artg4tk/package.py

```diff
@@ -44,6 +44,7 @@ class Artg4tk(CMakePackage):
     )
 
     patch("cetmodules2-c06a0ed7a0a543cba5c23fc588f7dd6dcb6609e2.patch", when="@c06a0ed7a0")
+    patch("11.00.01.patch", when="@11.00.01")
```

Comment:
Thanks for this, @yangtj207.  The version range is very specific (i.e. only version @11.00.01). Do you know if there was a change in the Geant4 interface?

Reply:
Hi @knoepfel, there was a bug in the old version of MyG4HadronPhysicsQGSP_BERT_ArHP.hh which Hans fixed in 2023 in his [commit](https://github.com/art-framework-suite/art-g4tk/commit/8dba20afe90772f68a09608f6c3e4181c22fc0d8#diff-e4049a44ae1fe5711bad459e57dec8f44157245d3cabf1ee8f0656f98ad50244). Currently this recipe for artg4tk uses 11.00.01 as the preferred version, which has this bug. After we move to a newer version of artg4tk, we won't need this patch. 
`Preferred version:  
    11.00.01       [git] https://github.com/art-framework-suite/art-g4tk.git at tag v11_00_01`